lms/templates/learner_dashboard/course_enroll_2017.underscore

```diff
@@ -1,4 +1,14 @@
-<% if (!is_enrolled) { %>
+<% if (is_enrolled && (typeof expired == 'undefined' || expired == false)) { %>
+    <% if (is_enrollment_open || is_course_ended) { %>
+        <a href="<%- course_url %>" class="view-course-button btn-brand btn cta-primary">
+            <% if (is_enrollment_open) { %>
+                <%- gettext('View Course') %>
+            <% } else if (is_course_ended) { %>
```

Comment:
What about the case where enrollment is closed but the course is still running? That's a possibility. If it happens, we still want to show "View Course."